flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/table/SavepointConnectorOptions.java

```diff
@@ -132,8 +133,27 @@ public enum StateType {
                     .stringType()
                     .noDefaultValue()
                     .withDescription(
-                            "Defines the format class scheme for decoding value data. "
-                                    + "When it's not provided then it tries to be inferred from the SQL type (only primitive types supported).");
+                            Description.builder()
+                                    .text(
+                                            "Defines the format class scheme for decoding value data. "
+                                                    + "Either %s or %s can be specified. "
+                                                    + "When none of them provided then it tries to be inferred from the SQL type (only primitive types supported).",
```

Comment:
nit: As is the  `provided` ->  `are provided`.

But I did not think it reads well. We are saying there are 2 things that can be provided, but if there is a case when they aren't.

I suggest 
 `%s or %s were not specified, so attempting to infer the format class scheme from the SQL type (only primitive types supported).`

Reply:
Thanks @gaborgsomogyi 

nit:
When none of them provided then it tries to be inferred from the SQL type 
->
When none of them are provided then it tries to infer from the SQL type

Reply:
Added.